EventVisualisation/DataConverter/include/EventVisualisationDataConverter/VisualisationEvent.h

```diff
@@ -66,7 +68,7 @@ class VisualisationEvent
     time_t collisionTime;
   };
   // Default constructor
-  VisualisationEvent(const VisualisationEventVO vo);
+  explicit VisualisationEvent(VisualisationEventVO vo);
```

Comment:
Should this have been passed by const reference ?

Reply:
done